document/CoreFormat.mdk

```diff
@@ -1120,11 +1185,78 @@ C++: `Namespace::Class::Method(int, double) const &&`
 C#: `Namespace1.Namespace2.Class.Method(System.String, int[])`
 ~
 
+If the `run.logicalLocations` property (&sect;[#run-logicalLocations]) is present,
+the value of the `fullyQualifiedLogicalName` property should be equal to the name of one of the properties on the `run.logicalLocations` object,
+with one exception, described in &sect;[#location-logicalLocationKey].
+
 ~ IsoNote
 The intent of this property is to allow a result log viewer to display the logical location
```

Comment:
Much but not all of this is already in the spec:

What's there:

* Logical location fills the gap when physical location is missing. I say that in NOTE 2 in Section Location-General.

* Intended for display. I say that in the NOTE in Section Location-FullyQualifiedLogicalName ("allows a result log viewer to display the location in a way that is easily understood by users")

* Helps viewers and humans to navigate. Also mentioned in the FQLN section, but I'll add "and other programs" to the now-existing text "code viewers and end users"

Here's what's not there:

* Intended for suppression. I'll add that.

* Relieves consumers of having to format it. I'll add that.

* Explaining why we have the array of logical location components (assist in organizing). I'll add that.